lib/vagrant-bhyve/driver.rb

```diff
@@ -146,6 +146,28 @@ def load_module(module_name)
 	end
       end
 
+      def check_or_create_default_pfconf(ui)
+        if execute(true, "test -s /etc/pf.conf") != 0
```

Comment:
Arguably, `/etc/pf.conf` should not be hardcoded here but maybe taken from the `pf_rules` `rcvar`.